### ISSUE TITLE
[Adjust Rule] SIM111 - Detect if another statement comes after the loop

### ISSUE THREAD

**Issue by kasium** [MartinThoma/flake8-simplify#62]
## Desired change

* **Rule(s)**: SIM111
* **Adjustment**: Detect if another statement comes after the loop

## Explanation

The below code is flagged with SIM111, but if I would would use the suggested code, the `call_method` call would be lost.

## Example

```python
for a in my_list:
  if a == 2:
    return False
call_method()
return True
```

**Comment by MartinThoma** [comment]
Sorry that this took so long. I'll make a release with this fix today :crossed_fingers:
